packages/fast-check-numbers/src/ZeroInt.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @since 1.1.0
+ */
+
+import * as fc from 'fast-check'
+import * as t from 'io-ts'
+import { Zero } from 'io-ts-numbers'
+
+/**
+ * @since 1.1.0
+ */
+export const ZeroIntArbitrary: fc.Arbitrary<Zero & t.Int> = fc
+  .oneof(fc.constant(0), fc.constant(-0))
+  .filter((n): n is Zero & t.Int => n === 0 && Number.isInteger(n))
```

Comment:
```suggestion
  .filter(ZeroInt.is)
```

Here I would prefer using the `ZeroInt` codec as a type-guard, to provide the guarantee that any value provided by the arbitrary will be decodable by the `ZeroInt` codec. This way, even if we get the logic in line 13 wrong, worst-case is we filter out the incorrect values and perhaps return a subset of the true `ZeroInt` value space.

Wait a minute -- zero is by definition an integer so isn't this entire codec moot? I think we can remove this entire file and just offer `Zero` via the package API. We can also remove the ZeroInt codec in io-ts-numbers